fuzz/fuzz_targets/deserialize_prefilled_transaction.rs

```diff
@@ -0,0 +1,61 @@
+extern crate bitcoin;
+
+fn do_test(data: &[u8]) {
+    // We already fuzz Transactions in `./deserialize_transaction.rs`.
+    let tx_result: Result<bitcoin::util::bip152::PrefilledTransaction, _> = bitcoin::consensus::encode::deserialize(data);
+
+    match tx_result {
+        Err(_) => {},
+        Ok(mut tx) => {
+            let ser = bitcoin::consensus::encode::serialize(&tx);
```

Comment:
I wonder how much would it help fuzzing to not have to allocate and just write to `&mut [u8]`, assuming we know the maximum length.

Reply:
Nice -- probably it'd be a nontrivial improvement.